frontend-react/src/components/DataDashboard/FacilitiesProviders/FacilitiesProviders.tsx

```diff
@@ -35,7 +35,7 @@ export function FacilitiesProviders() {
                     <hr />
                 </GridContainer>
             </header>
-            <GridContainer className="margin-left-7 margin-right-7 padding-top-4 padding-bottom-8 rs-max-width-100-important">
+            <GridContainer className="margin-left-7 margin-right-7 padding-top-4 rs-max-width-100-important">
```

Comment:
Can we find a way to avoid using `important` here? A locally scoped class within `_components.scss` if need be?

Reply:
Will do